HHS/Head-Start-TTADP
# [Prod] Print goals accounts for filters, rename/add new topics, fix for topics filter

## Description of change
- On the recipient record > goals & objectives tab, the "preview & print" button now accounts for the filters applied by the user.
- Topics were renamed; historical topic data was corrected
- Fix a bug where topics with parentheses were not being applied correctly as a filter to the goals

## How to test
- On the goals & objectives tab, try applying the filter "Goal topics" + "is" + "Curriculum (Instructional or Parenting)". This should filter correctly. [Here is a page with testable data](https://tta-smarthub-staging.app.cloud.gov/recipient-tta-records/1535/region/1/goals-objectives?).
- On that same page, you can test whether or not the filters are applied to the printing of goals. After you've filtered down, you can click "preview and print" and confirm that only the filtered goals appear in the print template.
- Check that the new topics are visible both on new report form controls and in historical reports ([details in this Jira](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-853))

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-853
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-916
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-980


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
